package.json

```diff
@@ -18,6 +18,7 @@
         "fixPrettierAll": "yarn prettier --write \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss,yml,php}\"",
         "fixPrettierChanged": "yarn pretty-quick --pattern \"**/*.{tsx,ts,jsx,js,json,md,html,css,scss,yml,php}\"",
         "refreshLando": "cd wordpress && lando refresh",
+        "runDbMigrations": "yarn typeorm migration:run",
```

Comment:
My only question for this PR:
We [used to have](https://github.com/owid/owid-grapher/blob/pre-v3/package.json#L19) `yarn knex migrate:latest && yarn typeorm migration:run` here. We don't use any Knex migrations any more, correct?

Reply:
Yes, correct!